### PR TITLE
ISSUE #2584 - Add GTM

### LINKED ISSUE #2584: Google Tag Manager for .io
### Description

We are now moving to Google Tag Manager, which allows us to embed a single tag reference and use it for a number of different products, including Google Analytics, without having to change backend code in the future

### Tasks

- [ ] Add the Google Tag Variable 'GTM-5H6W9JP'
- [ ] Add the code to .io
```
<!-- Google Tag Manager -->
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-5H6W9JP');</script>


<!-- End Google Tag Manager -->

```
Additionally, paste this code immediately after the opening <body> tag:

```
<!-- Google Tag Manager (noscript) -->
<noscript><iframe src=https://www.googletagmanager.com/ns.html?id=GTM-5H6W9JP
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<!-- End Google Tag Manager (noscript) -->

```

### PR DESCRIPTION
**Pull request by m1chalkubiak**
This fixes #2584

### CODE REVIEW

**Comment by keifgwinn** [comment]
I've made the tweaks required to deploy it on https://issue-2584.dev.3drepo.io/dashboard/teamspaces but I don't know if it's working or not.

**Comment by m1chalkubiak** [comment]
@keifgwinn I see code build with the newest changes, but don't see GTM added, are You sure gtmId is added on the server config for this branch ?

**Comment by keifgwinn** [comment]
root@issue-2584-98c6dd444-cjh7w:/home/node/3drepo.io/config/chart# printenv | grep GTM
APP_WEB_GTM_TRACK_ID=GTM-5H6W9JP
root@issue-2584-98c6dd444-cjh7w:/home/node/3drepo.io/config/chart# more config.js
<trimmed>
        ga: {
                trackId: process.env.APP_WEB_GA_TRACK_ID,
                refererTrackId: process.env.APP_WEB_GA_REFERER_TRACK_ID,
                refererDomain: process.env.APP_WEB_GA_REFERER_DOMAIN
        },
        gtm: {
                gtmId: process.env.APP_WEB_GTM_TRACK_ID, // "GTM-XXXXXX",
        },
 </trimmed>

File: frontend/services/analytics.ts [hunk 1]
```diff
@@ -1,118 +1,22 @@
...
-		if (!this.isGoogleAnalyticEnabled) {
-			return;
+			console.debug('Development - Not loading Google Analytics or remarketing');
 		}
 
-		if (tracker) {
-			ReactGA.pageview(location.pathname, [`${tracker}.send`]);
-		} else {
-			ReactGA.pageview(location.pathname);
+		if (!development) {
```
Comment (carmenfan) [inline frontend/services/analytics.ts:13]: this should just be else of the previous if condition?

**Review by carmenfan — CHANGES_REQUESTED** [review]

File: frontend/services/clientConfig.ts [hunk 2]
```diff
@@ -20,6 +20,7 @@ export class ClientConfigService {
 	// tslint:disable-next-line
 	public login_check_interval;
 	public ga;
```
Comment (carmenfan) [inline frontend/services/clientConfig.ts:22]: remove if we no longer use ga?

**Comment by m1chalkubiak** [comment]
It looks like it finally started working properly. @keifgwinn thanks for the help.

@carmenfan regarding the removal of `liveChatLicense` it looks like there is a component (`ReactLiveChat`) on FE that tries to use it. Perhaps this is something we should be looking at.

**Comment by m1chalkubiak** [comment]
Zrzut ekranu 2021-06-9 o 16 13 03

**Comment by carmenfan** [comment]
> It looks like it finally started working properly. @keifgwinn thanks for the help.
> 
> @carmenfan regarding the removal of `liveChatLicense` it looks like there is a component (`ReactLiveChat`) on FE that tries to use it. Perhaps this is something we should be looking at.



Can we get rid of the component? We haven't been using liveChat for the past 6 months now 😆  we moved over to intercom.

**Review by carmenfan — APPROVED** [review]

**Comment by carmenfan** [comment]
Ben verified that the tag is working.

**Pull request merged**